testsuite/tests/libratest/smoke_test.rs

```diff
@@ -454,10 +462,7 @@ fn test_basic_restartability() {
     let peer_to_restart = 0;
     // restart node
     env.validator_swarm.kill_node(peer_to_restart);
-    assert!(env
-        .validator_swarm
-        .add_node(peer_to_restart, RoleType::Validator, false)
-        .is_ok());
+    assert!(env.validator_swarm.add_node(peer_to_restart, false).is_ok());
```

Comment:
@davidiw, did you want to move away from the pattern "assert!(x.is_ok())", and instead do unwrap/expect? I know when these asserts fail, it can be difficult to debug

Reply:
Yeah, why not use unwrap?

Reply:
it's just copy-paste, I don't even look at the code lol